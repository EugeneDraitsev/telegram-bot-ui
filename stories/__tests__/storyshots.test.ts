// eslint-disable-next-line import/no-extraneous-dependencies
// import initStoryshots from '@storybook/addon-storyshots'
//
// initStoryshots({ /* configuration options */ })

describe('Placeholder', () => {
  test.todo('Re-enable initStoryshots when React 18 will be supported')
})
