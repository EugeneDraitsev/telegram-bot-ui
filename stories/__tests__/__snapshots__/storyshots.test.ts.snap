// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Buttons Buttons 1`] = `
<div
  className="sc-bdvvtL bgBQbm"
>
  <button
    className="sc-ksdxgE jRkPhr"
    style={
      Object {
        "marginRight": 10,
      }
    }
  >
    Default
  </button>
  <button
    className="sc-ksdxgE fkEYKe"
    disabled={true}
    style={
      Object {
        "marginRight": 10,
      }
    }
  >
    Disabled
  </button>
  <button
    className="sc-ksdxgE RwNbN"
    style={
      Object {
        "marginRight": 10,
      }
    }
  >
    Loading
  </button>
</div>
`;

exports[`Storyshots Chat Info Chat Info 1`] = `
<div
  className="sc-bdvvtL bgBQbm"
>
  <div
    className="sc-dkPtRN kqxILD"
  >
    <div
      className="sc-hKwDye iuxoy"
    >
      <div
        className="sc-eCImPb kSBlNu"
        src="https://chat-profile-images.s3.amazonaws.com/file_70280.jpg"
      />
      <div
        className="sc-jRQBWg fqWysv"
      >
        Kabold —Åamp
        <div
          className="sc-gKclnd gefYSt"
        >
          The kaboldest camp
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Chat Info Edge Case Chat Info 1`] = `
<div
  className="sc-bdvvtL bgBQbm"
>
  <div
    className="sc-dkPtRN kqxILD"
  >
    <div
      className="sc-hKwDye iuxoy"
    >
      <div
        className="sc-eCImPb kpRPGw"
        src="/favicon.png"
      />
      <div
        className="sc-jRQBWg fqWysv"
      >
        Unknown Chat
        <div
          className="sc-gKclnd gefYSt"
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Graphs Bar Chart 1`] = `
<div
  className="sc-bdvvtL bgBQbm"
>
  <div
    className="sc-bkkeKt jJmDdv"
  >
    <div
      className="recharts-responsive-container"
      style={
        Object {
          "height": 400,
          "maxHeight": undefined,
          "minHeight": undefined,
          "minWidth": undefined,
          "width": "100%",
        }
      }
    />
    <div
      className="sc-ieecCq dCvCmi"
    >
      <div
        className="sc-dJjYzT hAupCo"
      >
        <div
          className="sc-hGPBjI cchtBy"
          color="#1f77b4"
        />
        <div
          className="sc-dlVxhl iYsQSU"
        >
          user1  (62)
        </div>
      </div>
      <div
        className="sc-dJjYzT hAupCo"
      >
        <div
          className="sc-hGPBjI jfhueX"
          color="#ff7f0e"
        />
        <div
          className="sc-dlVxhl iYsQSU"
        >
          user2  (52)
        </div>
      </div>
      <div
        className="sc-dJjYzT hAupCo"
      >
        <div
          className="sc-hGPBjI gRYBDG"
          color="#2ca02c"
        />
        <div
          className="sc-dlVxhl iYsQSU"
        >
          user3  (30)
        </div>
      </div>
      <div
        className="sc-dJjYzT hAupCo"
      >
        <div
          className="sc-hGPBjI kZWDtS"
          color="#d62728"
        />
        <div
          className="sc-dlVxhl iYsQSU"
        >
          user4  (27)
        </div>
      </div>
      <div
        className="sc-dJjYzT hAupCo"
      >
        <div
          className="sc-hGPBjI buvhLB"
          color="#9467bd"
        />
        <div
          className="sc-dlVxhl iYsQSU"
        >
          user5  (16)
        </div>
      </div>
      <div
        className="sc-dJjYzT hAupCo"
      >
        <div
          className="sc-hGPBjI jDtVod"
          color="#8c564b"
        />
        <div
          className="sc-dlVxhl iYsQSU"
        >
          user6  (10)
        </div>
      </div>
      <div
        className="sc-dJjYzT hAupCo"
      >
        <div
          className="sc-hGPBjI bvoac"
          color="#e377c2"
        />
        <div
          className="sc-dlVxhl iYsQSU"
        >
          user7  (9)
        </div>
      </div>
      <div
        className="sc-dJjYzT hAupCo"
      >
        <div
          className="sc-hGPBjI dXMvti"
          color="#7f7f7f"
        />
        <div
          className="sc-dlVxhl iYsQSU"
        >
          user8  (5)
        </div>
      </div>
      <div
        className="sc-dJjYzT hAupCo"
      >
        <div
          className="sc-hGPBjI bTJQfE"
          color="#bcbd22"
        />
        <div
          className="sc-dlVxhl iYsQSU"
        >
          user9  (3)
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Graphs Pie Chart 1`] = `
<div
  className="sc-bdvvtL bgBQbm"
>
  <div
    className="sc-hBUSln iRZdkq"
  >
    <div
      className="recharts-responsive-container"
      style={
        Object {
          "height": "100%",
          "maxHeight": undefined,
          "minHeight": undefined,
          "minWidth": undefined,
          "width": "100%",
        }
      }
    />
  </div>
</div>
`;

exports[`Storyshots Input Icon Input 1`] = `
<div
  className="sc-bdvvtL bgBQbm"
>
  <div
    className="sc-llYSUQ bUVtJk"
  >
    <div>
      <div
        className="sc-crHmcD fsrLkM"
      >
        <input
          className="sc-bqiRlB lgpFNM"
          value="Simple input"
        />
        <div
          className="sc-egiyK beyLVh"
        >
          <svg
            fill="none"
            height={24}
            stroke="currentColor"
            strokeLinecap="round"
            strokeLinejoin="round"
            strokeWidth="2"
            viewBox="0 0 24 24"
            width={24}
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M9 19c-5 1.5-5-2.5-7-3m14 6v-3.87a3.37 3.37 0 0 0-.94-2.61c3.14-.35 6.44-1.54 6.44-7A5.44 5.44 0 0 0 20 4.77 5.07 5.07 0 0 0 19.91 1S18.73.65 16 2.48a13.38 13.38 0 0 0-7 0C6.27.65 5.09 1 5.09 1A5.07 5.07 0 0 0 5 4.77a5.44 5.44 0 0 0-1.5 3.78c0 5.42 3.3 6.61 6.44 7A3.37 3.37 0 0 0 9 18.13V22"
            />
          </svg>
        </div>
      </div>
    </div>
    <div>
      <div
        className="sc-crHmcD fsrLkM"
      >
        <input
          className="sc-bqiRlB lgpFNM"
          value="Emoji ü¶Ñ"
        />
        <div
          className="sc-egiyK beyLVh"
        >
          üêà
        </div>
      </div>
    </div>
    <div>
      <div
        className="sc-crHmcD fsrLkM"
      >
        <input
          className="sc-bqiRlB enIqlQ"
          style={
            Object {
              "width": 215,
            }
          }
          value="123,43"
        />
        <div
          className="sc-egiyK beyLVh"
        >
          $
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Input Simple Input 1`] = `
<div
  className="sc-bdvvtL bgBQbm"
>
  <div
    className="sc-crHmcD fsrLkM"
  >
    <input
      className="sc-bqiRlB fsQPaS"
      value="Simple input"
    />
  </div>
</div>
`;

exports[`Storyshots Spinners Spinners 1`] = `
<div
  className="sc-bdvvtL bgBQbm"
>
  <svg
    aria-label="spinner"
    className="sc-iCfMLu kFTwcL"
    style={
      Object {
        "height": 50,
        "width": 50,
      }
    }
    viewBox="25 25 50 50"
  >
    <circle
      className="sc-furwcr loMGra"
      cx="50"
      cy="50"
      fill="none"
      r="20"
      strokeMiterlimit="10"
      strokeWidth={2}
    />
  </svg>
  <svg
    aria-label="spinner"
    className="sc-iCfMLu kFTwcL"
    style={
      Object {
        "height": 105,
        "width": 105,
      }
    }
    viewBox="25 25 50 50"
  >
    <circle
      className="sc-furwcr loMGra"
      cx="50"
      cy="50"
      fill="none"
      r="20"
      strokeMiterlimit="10"
      strokeWidth={2}
    />
  </svg>
  <svg
    aria-label="spinner"
    className="sc-iCfMLu kFTwcL"
    style={
      Object {
        "height": 50,
        "width": 50,
      }
    }
    viewBox="25 25 50 50"
  >
    <circle
      className="sc-furwcr dnPqdt"
      color="red"
      cx="50"
      cy="50"
      fill="none"
      r="20"
      strokeMiterlimit="10"
      strokeWidth={2}
    />
  </svg>
  <svg
    aria-label="spinner"
    className="sc-iCfMLu kFTwcL"
    style={
      Object {
        "height": 100,
        "width": 100,
      }
    }
    viewBox="25 25 50 50"
  >
    <circle
      className="sc-furwcr kedQsF"
      color="green"
      cx="50"
      cy="50"
      fill="none"
      r="20"
      strokeMiterlimit="10"
      strokeWidth={2}
    />
  </svg>
</div>
`;

exports[`Storyshots Tabs Simple Tabs 1`] = `
<div
  className="sc-bdvvtL bgBQbm"
>
  <div
    className="sc-iJKOTD exqLpx"
  >
    <div
      className="sc-pVTFL cPesfZ"
    >
      <div
        className="sc-jrQzAO dscZjF"
      >
        <div
          className="sc-kDTinF jBRJHx tab active"
          onClick={[Function]}
        >
          Tab1
        </div>
        <div
          className="sc-kDTinF jBRJHx tab"
          onClick={[Function]}
        >
          Veeeeeery Loooooong Name Tab
        </div>
        <div
          className="sc-kDTinF jBRJHx tab"
          onClick={[Function]}
        >
          Some other tab
        </div>
        <div
          className="sc-kDTinF jBRJHx tab"
          onClick={[Function]}
        >
          tab 4
        </div>
        <div
          className="sc-iqseJM euGiBn"
          tabIndex={0}
          width={0}
          x={0}
        />
      </div>
    </div>
    <div
      className="sc-giYglK egAvaJ"
    >
      selected 
      Tab1
       tab!
    </div>
  </div>
</div>
`;
